cmd/kubelet/app/options/options.go

```diff
@@ -533,4 +533,6 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.BoolVar(&c.RegisterNode, "register-node", c.RegisterNode, "Register the node with the apiserver. If --kubeconfig is not provided, this flag is irrelevant, as the Kubelet won't have an apiserver to register with.")
 
 	fs.Var(&utilflag.RegisterWithTaintsVar{Value: &c.RegisterWithTaints}, "register-with-taints", "Register the node with the given list of taints (comma separated \"<key>=<value>:<effect>\"). No-op if register-node is false.")
+
+	fs.DurationVar(&c.PLEGRelistThreshold.Duration, "pleg-relist-threshold", c.PLEGRelistThreshold.Duration, "The threshold for PLEG relisting, default 3m.")
```

Comment:
Many kubelet flags are migrating into the [kubelet configuration yaml](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/). This may be not needed.

Reply:
if not need. I will remove this flags. @pacoxu